### PR TITLE
[Beloin](feature) Added Tasks to Launch.nvim

### PR DESCRIPTION
**Pull request by Beloin**
- Added request atribute
- Updated README.md
- Added examples
- Added TODOS to README.md
- Renamed Launch command to `Launch[Action]`
- Using Picker and Previewer to `LaunchTasks`
  - Added lua as type
- Loading tasks inside launch

### CODE REVIEW

File: lua/Launch/Tasks/init.lua [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	launch_parser.load()
+
+	local task_table = launch_parser.get_tasks()
+	if not task_table then
+		utils.notify_error("Problems loading task table: " .. task_table)
+		return false
+	end
+
+	local has_tasks = tasks_parser.load_tasks(task_table)
+	if not has_tasks then
```
Comment (arthurqueiroz4) [inline lua/Launch/Tasks/init.lua:23]: sempre tem que ter tasks no launch.nvim?

File: lua/Launch/Tasks/picker.lua [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+		define_preview = function(self, entry, _)
+			-- Clear the buffer first
+			vim.api.nvim_buf_set_lines(self.state.bufnr, 0, -1, false, {})
+
+			local content = entry.value.preview or "No preview available"
+			vim.api.nvim_buf_set_lines(self.state.bufnr, 0, -1, false, vim.split(content, "\n"))
+
+      -- Configure Buffer information
+      vim.bo[self.state.bufnr].filetype = "lua"
+      vim.bo[self.state.bufnr].modifiable = false
```
Comment (arthurqueiroz4) [inline lua/Launch/Tasks/picker.lua]: ajustar indentação

File: lua/Launch/Tasks/init.lua (hunk 1 above)
Comment (Beloin) [reply lua/Launch/Tasks/init.lua:23]: Se não tiver e o indivíduo quiser rodar "LaunchTasks" aí temos um problema

File: lua/Launch/Tasks/picker.lua (hunk 2 above)
Comment (Beloin) [reply lua/Launch/Tasks/picker.lua]: Done

**Review by arthurqueiroz4 — APPROVED** [review]

**Pull request merged**